src/cryptonote_core/cryptonote_core.cpp

```diff
@@ -1230,6 +1230,7 @@ void core::start_oxenmq() {
                 false,
                 m_pulse_thread_id);
         m_omq->add_timer([this]() { this->check_service_node_time(); }, 5s, false);
+        m_omq->add_timer([this]() { this->check_service_node_ip_address(); }, 900s, false);
```

Comment:
`900s` -> `15min`

Reply:
This was throwing an error in my CLion IDE with 15m but I'll look into it